src/main/scala/com/typesafe/sbt/packager/archetypes/jlink/JlinkKeys.scala

```diff
@@ -0,0 +1,21 @@
+package com.typesafe.sbt.packager.archetypes
+package jlink
+
+import sbt._
+
+import com.typesafe.sbt.packager.archetypes.jlink._
+
+/**
+  * Available settings/tasks for the [[com.typesafe.sbt.packager.archetypes.jlink.JlinkPlugin]].
+  */
+trait JlinkKeys {
```

Comment:
We are starting to make these traits private. This helps in keeping things binary compatible

Reply:
Made this `private[packager]`.

Reply:
sweet